config/settings/test.yml

```diff
@@ -204,6 +204,13 @@ sidekiq:
   github_oauth_secret: 000xxx
 
 check_in:
```

Comment:
Are any of these settings not used for tests? If so, you'll have to create a corresponding [devops](https://github.com/department-of-veterans-affairs/devops/tree/master/ansible/deployment/config/vets-api) PR for the `settings.yml` for each environment.

Reply:
Yes, we are handling adding configs in devops repo in a separate ticket: https://github.com/department-of-veterans-affairs/va.gov-team/issues/47014